.travis/build.sh

```diff
@@ -11,6 +11,18 @@ exitIfError() {
   [ "$EXIT" != "0" ] && exit $EXIT
 }
 
+# # # # # # # # # # # # # # # # # # # # # # #
+#
+#   These tests are now disabled by default.
+#
+#
+
+if [ "$DISABLED" != "false" ]; then
+  echo "Tests are disabled by default. In order to run them locally set DISABLED=false"
+  echo "Usage: DISABLED=false ./build.sh"
+  exit 0
+fi
+
```

Comment:
why aren't we just removing the `.travis` file? Didn't this proposal have the purpose of doing that? https://github.com/strimzi/proposals/blob/main/099-drop-travis-ci-and-testing-for-ppc-and-s390x.md

Reply:
I can do that as well yes.

Reply:
yes, `.travis` shoudl be removed.